google_maps/public/html/google_maps_edit.html

```diff
@@ -1,15 +1,67 @@
-<div class="wrapper-comp-settings is-active editor-with-buttons google-maps-edit-wrapper">
-  <p>Welcome to Google Maps Xblock!</p>
+{% load i18n %}
+
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="google-maps-edit-wrapper">
+  <div class="xblock-inputs editor_content_wrapper">
+    <ul class="list-input settings-list">
+      <li class="field comp-setting-entry is-set edit-display-name-container">
+        <div class="wrapper-comp-setting">
+          <label class="label setting-label" for="edit_display_name">{{ fields.display_name.display_name }}</label>
+          <input class="input setting-input edit-display-name" id="edit_display_name" value="{{ self.display_name }}" type="text" 
+          data-default-value="{{ fields.display_name.default }}">
```

Comment:
a11y:

add the following attribute to the input element `aria-describedby="display-name-help"`. Span element below contains additional information that screen reader users might find helpful.